homeassistant/components/litejet/light.py

```diff
@@ -91,16 +93,37 @@ def extra_state_attributes(self):
 
     def turn_on(self, **kwargs):
         """Turn on the light."""
+        is_complex = False
+        brightness = 99
+        transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
+
         if ATTR_BRIGHTNESS in kwargs:
+            is_complex = True
             brightness = int(kwargs[ATTR_BRIGHTNESS] / 255 * 99)
-            self._lj.activate_load_at(self._index, brightness, 0)
+
+        if ATTR_TRANSITION in kwargs:
+            is_complex = True
+            transition = kwargs[ATTR_TRANSITION]
+
+        if is_complex:
```

Comment:
It is always a "complex" call, right? As there is a default transition?

Reply:
It is complicated... The LiteJet system internally has a default transition per-light that HA can't access. LiteJet uses this internal value only for the simple call of on or off without any parameters. If brightness is specified then a transition also must be specified to LiteJet. Think of the HA default as a fallback default to be used only when a transition value must be specified from the HA side.

I can comment this to be more clear to future maintainers.

Reply:
In this case, a comment might help.

Reply:
Alternative suggestion:

```py
        if ATTR_BRIGHTNESS not in kwargs and ATTR_TRANSITION not in kwargs:
           self._lj.activate_load(self._index)
           return

        default_transition = self._config_entry.options.get(CONF_DEFAULT_TRANSITION, 0)
        transition = kwargs.get(ATTR_TRANSITION, default_transition)
        brightness = kwargs.get(ATTR_BRIGHTNESS, 99)
        
        self._lj.activate_load_at(self._index, brightness, int(transition))
```